ai_workloads/kueue/monitoring-pending-workloads.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="monitoring-pending-workloads-install-kueue"]
+= Monitoring pending workloads 
+:context: monitoring-pending-workloads
+
+toc::[]
+
+[role="_abstract"]
+{kueue-name} provides the `VisibilityOnDemand` feature to monitor pending workloads. A workload is an application that runs to completion. It can be composed by one or multiple pods that, loosely or tightly coupled, as a whole, complete a task. A workload is the unit of admission in {kueue-name}.
+
+The `VisibilityOnDemand` feature provides the ability for batch administrators to monitor the pipeline of pending jobs in the cluster queue and the local queue and batch users just for local queue, and help users to estimate when their jobs will start.
+
+You can regulate inbound requests and high request volumes, and provide user permissions for viewing the pending workloads.
+
+.Prerequisites 
+
+* You have cluster administrator permissions.
+* The {kueue-name} Operator is installed on your cluster, and you have created a `Kueue` custom resource (CR).
```

Comment:
This needs to be moved into the procedure module. It is throwing a vale dita error. https://spaces.redhat.com/pages/viewpage.action?pageId=654185635&spaceKey=OSDOCS&title=Content%2Bupdates%2Bin%2Bpreparation%2Bfor%2Bmigrating%2Bto%2BDITA#:~:text=Test%20admonition%20text.%0A%3D%3D%3D%3D-,AssemblyContents,-After%20the%20first